server/src/main/resources/static/open-api.yaml

```diff
@@ -90,6 +90,68 @@ paths:
         500:
           description: "Virheellinen pyyntö"
 
+  # Mock API for YKI suoritukset
+  /dev/yki/arvioijat:
+    get:
+      tags:
+        - "yki-controller"
+      summary: "Hakee YKI arvioijat"
+      operationId: "getArvioijat"
+      responses:
+        200:
+          description: "Arvioija tai arvioijia löytyi"
+          content:
+            "text/plain":
+              schema:
+                type: string
+
+  # Mock API for YKI suoritukset
+  /dev/yki/suoritukset:
+    get:
+      tags:
+        - "yki-controller"
+      summary: "Hakee YKI suoritukset"
+      operationId: "getSuoritukset"
+      parameters:
+        - name: arvioitu
+          in: query
+          description: "Arviointi päivämäärä"
+          required: false
+          content:
+            "text/plain":
+              schema:
+                type: string
+                format: date
+
+      responses:
+        200:
+          description: "Suoritus tai suorituksia löytyi"
+          content:
+            "text/plain":
+              schema:
+                type: string
+
+  /dev/yki/trigger-schedule:
```

Comment:
Pitäisikö näitä nimetä käytänteellä `/dev/<domain>/trigger-schedule/<suoritettavan tehtävän nimi>` siltä varalta että johonkin domainiin liittyy useita ajastettuja tehtäviä?

Reply:
Kokeilin ottaa käyttöö spring-bootin `actuator/scheduledtasks`, mutta meidän schedulerit ei näy siellä. Sit taas meidän käyttämä task-scheduler kirjasto [suosittelee](https://github.com/kagkarlsson/db-scheduler?tab=readme-ov-file#third-party-extensions) eri [työkalua](https://github.com/bekk/db-scheduler-ui) schedulointiin.